kitsune/search/tests/test_api.py

```diff
@@ -122,15 +106,15 @@ def test_invalid_numbers(self):
             'q': 'search',
         })
         eq_(res.status_code, 400)
-        eq_(res.data['detail'], {
-            'max_questions': [u'Enter a whole number.'],
-            'max_documents': [u'Enter a whole number.'],
+        eq_(res.data, {
```

Comment:
This suggests there's a change in the API results. It looks like it's entirely with the "something is wrong with the request" case and that the "everything is fine--here's your search results" case is ok.

Is that true?